### PR TITLE
Readded playlist to like action from guglielmobartelloni:playlist-to-like

### LINKED ISSUE #94: [Feature Request] Like Songs in YouTube music
Option for adding imported songs from Spotify to be added to "Liked Music" playlist of YouTube music instead of "Liked Music (Spotify)". Meaning like songs as they getting imported. All this with the `spotify_to_ytmusic liked` option.

### PR DESCRIPTION
**Pull request by Tim-Odermatt**
- Added parameters to like all songs for a single converted playlist
- Added parameters to like all songs for all public converted playlists of a users
- Updated README

closes #94
closes #38

### CODE REVIEW

**Comment by Tim-Odermatt** [comment]
Would close #94

File: spotify_to_ytmusic/controllers.py [hunk 1]
```diff
@@ -46,6 +46,10 @@ def all(args):
                 "PUBLIC" if p["public"] else "PRIVATE",
                 videoIds,
             )
+            if args.like:
+                for id in videoIds:
+                    ytmusic.rate_song(id, "LIKE")
+                    print(id)
```
Comment (sigma67) [inline spotify_to_ytmusic/controllers.py:52]:
```suggestion
```

File: spotify_to_ytmusic/controllers.py [hunk 2]
```diff
@@ -58,6 +62,10 @@ def _create_ytmusic(args, playlist, ytmusic):
     name = args.name + date if args.name else playlist["name"] + date
     info = playlist["description"] if (args.info is None) else args.info
     videoIds = ytmusic.search_songs(playlist["tracks"])
+    if args.like:
+        for id in videoIds:
+            ytmusic.rate_song(id, "LIKE")
+            print(id)
```
Comment (sigma67) [inline spotify_to_ytmusic/controllers.py:68]:
```suggestion
```

**Comment by sigma67** [comment]
@Tim-Odermatt use this syntax to correctly link :)

closes #94
closes #38

**Pull request merged**